components/infobox/commons/infobox_league.lua

```diff
@@ -282,6 +282,8 @@ function League:_parseArgs()
 
 	self.data = data
 
+	self.data.publishertier = tostring(Logic.readBool(args.publisherpremier))
```

Comment:
Could be moved into the initial data definition (line 277)

Reply:
have moved this one in the newest revision now